docs/release-notes/release-notes-0.19.2.md

```diff
@@ -0,0 +1,115 @@
+    - [Functional Updates](#functional-updates)
+    - [RPC Updates](#rpc-updates)
+    - [lncli Updates](#lncli-updates)
+    - [Breaking Changes](#breaking-changes)
+    - [Performance Improvements](#performance-improvements)
+    - [Deprecations](#deprecations)
+- [Technical and Architectural Updates](#technical-and-architectural-updates)
+    - [BOLT Spec Updates](#bolt-spec-updates)
+    - [Testing](#testing)
+    - [Database](#database)
+    - [Code Health](#code-health)
+    - [Tooling and Documentation](#tooling-and-documentation)
+
+# Bug Fixes
+
+- [Use](https://github.com/lightningnetwork/lnd/pull/9889) `BigSizeT` instead of
+  `uint16` for the htlc index that's used in the revocation log.
+
+- [Fixed](https://github.com/lightningnetwork/lnd/pull/9921) a case where the
+  spending notification of an output may be missed if wrong height hint is used.
+
+- [Fixed](https://github.com/lightningnetwork/lnd/pull/9962) a case where the
+  node may panic if it's running in the remote signer mode.
+
+- [Fixed the `historical channel bucket has not yet been created` error on
+  startup](https://github.com/lightningnetwork/lnd/pull/9653).
+
+- [Fixed](https://github.com/lightningnetwork/lnd/pull/9978) a deadlock which
+  can happen when the peer start-up has not yet completed but a another p2p
+  connection attempt tries to disconnect the peer.
+  
+- [Fixed](https://github.com/lightningnetwork/lnd/pull/10012) a case which
+  could lead to a memory issues due to a goroutine leak in the peer/gossiper
+  code.
+
+# New Features
+
+## Functional Enhancements
+
+- [Adds](https://github.com/lightningnetwork/lnd/pull/9989) a method 
+  `FeeForWeightRoundUp` to the `chainfee` package which rounds up a calculated 
+  fee value to the nearest satoshi.
+
+- [Update](https://github.com/lightningnetwork/lnd/pull/9996) lnd to point at
+  the new deployment of the lightning seed service which is used to provide
+  candidate peers during initial network bootstrap. The lseed service now
+  supports `testnet4` and `signet` networks as well.
+
+## RPC Additions
+
+* When querying
+[`ForwardingEvents`](https://github.com/lightningnetwork/lnd/pull/9813) logs,
+the response now include the incoming and outgoing htlc indices of the payment
+circuit. The indices are only available for forwarding events saved after v0.20.
+
+## lncli Additions
+
+# Improvements
+
+## Functional Updates
+
+- [Improved](https://github.com/lightningnetwork/lnd/pull/9880) the connection
+  restriction logic enforced by `accessman`. In addition, the restriction placed
+  on outbound connections is now lifted.
+- [Enhanced](https://github.com/lightningnetwork/lnd/pull/9980) the aux traffic
+  shaper to now accept the first hop peer pub key as an argument. This can
+  affect the reported aux bandwidth and also the custom records that are
+  produced.
+## RPC Updates
+
+## lncli Updates
+
+## Code Health
+
+- [Add Optional Migration](https://github.com/lightningnetwork/lnd/pull/9945)
```

Comment:
Probably worth noting when we publish the release the the `sphinxreplay.db` memory requirements are significantly reduced.